### PR TITLE
Feature/com 210 com 211 spawn second car change meatspin

### PR DESCRIPTION
**Pull request by Lukauigi**
Car Selection Functionality!
* Garage Controller entity with script
* 2nd Car added
* Car Selection Meatspin (spacebar)
* Choose Car (enter key)
* Unbind controls after car is chosen

### CODE REVIEW

**Review by jharris99-git — APPROVED** [review]
Functionality there.

- Fix car textures.
- Wait for @le-rich or @imnorookie approval for merging, for style.

**Review by imnorookie — APPROVED** [review]
Looks good to me.

File: Physics/include/Body.h [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #pragma once
-#ifndef BODY_HEADER
+//#ifndef BODY_HEADER
```
Comment (le-rich) [inline Physics/include/Body.h:2]: Remove dead code

File: Physics/Physics.h [hunk 2]
```diff
@@ -6,74 +8,65 @@
...
+        std::vector<UUIDv4::UUID> rigidBodyEntitiesUUIDs = EntityManager::getInstance().findEntitiesByComponent(ComponentType::RigidBody);
+
+        for (UUIDv4::UUID rigidBodyEntityUUID : rigidBodyEntitiesUUIDs) {
+            Entity* rigidBodyEntity = EntityManager::getInstance().getEntity(rigidBodyEntityUUID);
+            RigidBodyComponent* rigidBodyComponent = dynamic_cast<RigidBodyComponent*>(
+                rigidBodyEntity->getComponent(ComponentType::RigidBody)
+                );
+
+            //    AE86::RigidBody* rigidBody = rigidBodyComponent->getRigidBody().get();
+             //   addBody(rigidBodyComponent);
```
Comment (le-rich) [inline Physics/Physics.h:35]: Remove dead code

**Review by le-rich — APPROVED** [review]

**Pull request merged**